src/resources/postcss/tickets-attendees-print.pcss

```diff
@@ -51,11 +49,35 @@
 	}
 
 	#tribe-attendees-summary .welcome-panel-column {
-		width: 100%;
 		padding: 0;
 		float: none;
 	}
 
+	#tribe-attendees-summary {
+		margin-bottom: 30px;
+		.welcome-panel-column {
```

Comment:
yes, yes...I know and preach this very thing! A sure sign I need to take a nap! ;)